src/Sulu/Bundle/ContentBundle/Command/MaintainResourceLocatorCommand.php

```diff
@@ -136,7 +135,14 @@ private function upgradeNode(
 
             // save value
             $property->setValue($rl);
-            $resourceLocator->write($node, $transProperty, 1, $webspace->getKey(), $localization->getLocalization());
+            $resourceLocator->write(
+                $node,
+                $transProperty,
+                1,
+                $webspace->getKey(),
+                $localization->getLocalization(),
+                null
+            );
             $session->save();
```

Comment:
I am not even sure, and this is out of scope of this PR anyway. I was just making sure that this command is still working :-)